treeherder/model/models.py

```diff
@@ -1074,7 +1075,8 @@ class FailureLine(models.Model):
     status = models.CharField(max_length=7, choices=STATUS_CHOICES)
     expected = models.CharField(max_length=7, choices=STATUS_CHOICES, blank=True, null=True)
     message = models.TextField(blank=True, null=True)
-    signature = models.TextField(blank=True, null=True)  # Prefix index length 50
+    # Prefix index: signature(50)
+    signature = models.TextField(blank=True, null=True, db_index=True)
```

Comment:
What does `db_index=True` do in this case, given that we're creating a prefix index with custom SQL?